src/frontend/src/lib/types/network.ts

```diff
@@ -10,6 +11,7 @@ export interface Network {
 	env: NetworkEnvironment;
 	name: string;
 	icon?: string;
+	iconBW?: ComponentType;
```

Comment:
- I'm pretty that I will forgot what `BW` means I have to say.
- I was also wondering if we should, in another PR, group the icons?

```
env: ...
name: string;
icons: NetworkIcons;

type NetworkIcons {
   primary: ComponentType;
   negative: ComponentType;
}
```

(naming as example)

Reply:
Yes, but i agree, another PR, so i change the name directly there, WDYT?

Reply:
Another PR 👍